### PR TITLE
add: `f1multiviewer-deb`

### PR DESCRIPTION
**Pull request by DismissedGuy**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

File: packages/f1multiviewer-deb/f1multiviewer-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name="f1multiviewer-deb"
+pkgname="f1multiviewer"
```
Comment (Henryws) [inline packages/f1multiviewer-deb/f1multiviewer-deb.pacscript:2]:
```suggestion
gives="f1multiviewer"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by DismissedGuy** [comment]
Does `gives` break it somehow?

File: packages/f1multiviewer-deb/f1multiviewer-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+gives="${pkgname}"
+repology=("project: ${pkgname}")
+version="1.22.2"
+maintainer="DismissedGuy <me@mikealmel.ooo>"
+homepage="https://multiviewer.app/"
+url="https://releases.multiviewer.app/download/113292497/multiviewer-for-f1_${pkgver}_amd64.deb"
```
Comment (Henryws) [inline packages/f1multiviewer-deb/f1multiviewer-deb.pacscript:8]:
```suggestion
url="https://releases.multiviewer.app/download/113292497/multiviewer-for-f1_${version}_amd64.deb"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/f1multiviewer-deb/f1multiviewer-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name="f1multiviewer-deb"
+pkgname="f1multiviewer"
+gives="${pkgname}"
```
Comment (Henryws) [inline packages/f1multiviewer-deb/f1multiviewer-deb.pacscript:3]:
```suggestion
gives="multiviewer-for-f1"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**